app/views/assessment/general_questions/_form.html.erb

```diff
@@ -5,6 +5,34 @@
     <%= f.input :max_grade, as: :integer, input_html: {class: "number_field input-mini"}, min: 0  %>
     <%= f.input :tags, as: :tag, url: course_tags_path, input_html: {class: "span7"}  %>
 
+    <%= f.input :auto_graded,
+        input_html: {"data-toggle" => "collapse", "data-target" => "#auto-grading-type"} %>
+
+    <div id="auto-grading-type" class="collapse <%= @question.auto_graded? ? "in" : "out" %>">
+
+        <%= f.input :auto_grading_type,
+            as: :radio_buttons,
+            label: 'Auto-grading type',
+            collection: [ ['None', 0], ['Exact match', 1], ['Keywords', 2]],
```

Comment:
Rails 3 doesn't support ActiveRecord.enum. What's the preferred way to express this?